go-controller/pkg/util/sriovnet_linux.go

```diff
@@ -134,6 +144,8 @@ func GetNetdevNameFromDeviceId(deviceId string) (string, error) {
 		}
 
 		netdevices, err = GetSriovnetOps().GetNetDevicesFromPci(deviceId)
+	} else { // Auxiliary network device
```

Comment:
It's a bit confusing but is Auxiliary a superset of SFs? Does that mean there could be other types of SF-like interfaces? Or is Auxiliary == SF?

Reply:
You mention SF in commits and commentary; however the code function names are all named Auxiliary which is a bit confusing. Maybe add in an explanation of how Auxiliary relates to SF in code somewhere and I think it should be good.

Reply:
Yes, SF is a subset of Auxiliary devices, just like VF is a subset of PCI devices. AFAIK, only SF implements representor model.

Reply:
> Maybe add in an explanation ..

In You opinion, what is the best place for it?